src/Core/Profiling/Subscriber/HttpCacheTagDataCollectorSubscriber.php

```diff
@@ -40,25 +42,28 @@ public function store(HttpCacheStoreEvent $event): void
     {
         $uri = $this->uri($event->request);
 
-        self::$tags[$uri] = $event->tags;
+        foreach ($event->tags as $tag) {
```

Comment:
just curious, where does this method get called?

Reply:
good question. it seems to be unused. I removed it. let's look, if something explodes :grin: